oasis-reference/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Thinking about creating a task to do this for us, however, I am not sure how will this work with intellij test runner if it is in gradle when we are not using grader build(or are we?) when you click on run/run test button.

This is added base on this [release note about mock the unmockable](https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#unmockable) as suggested in this [post](https://trickyandroid.com/using-mockito-with-kotlin/)

Reply:
As we chatted about powermock, I do want to playing around with both once #12 is in since it also involve some refactored UI components.